inbm/dispatcher-agent/tests/unit/schedule/test_sqlite_manager.py

```diff
@@ -53,22 +56,28 @@ def test_raise_exception_when_create_repeated_schedule_with_no_manifests(db_conn
 
 def test_create_simple_schedule(db_connection: SqliteManager):
     db_connection.clear_database()
-    ss1 = SingleSchedule(request_id="REQ123",
+    ss1 = SingleSchedule(request_id="4324a262-b7d1-46a7-b8cc-84d934c3983f",
+                         job_id="swupd-939fe48c-32da-40eb-a00f-acfdb43a5d6d",
                          start_time=datetime.strptime("2024-01-01T00:00:00", "%Y-%m-%dT%H:%M:%S"),
                          end_time=datetime.strptime("2024-01-02T00:00:00", "%Y-%m-%dT%H:%M:%S"),
                          manifests=["MANIFEST1", "MANIFEST2"])
```

Comment:
**<h2>$\color{#15F4EE}Miscellaneous$</h2>**
**Suggestion:** Implement a method in SqliteManager to validate manifest content before insertion.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
                         def validate_manifests(self, manifests):
                             for manifest in manifests:
                                 if not self.is_valid_manifest(manifest):
                                     raise DispatcherException("Invalid manifest content")
                         validate_manifests(manifests=["MANIFEST1", "MANIFEST2"])
```

Reply:
Manifests are validated against the TC schema before being added to the database or ran if immediate.